docs/cloud-workload-security/secl.json

```diff
@@ -313,6 +313,12 @@
           "definition": "Group of the process",
           "constants": ""
         },
+        {
+          "name": "process.ancestors.is_kworker",
+          "type": "bool",
+          "definition": "",
```

Comment:
As above, could you confirm if it's expected for `definition` to be an empty string here, as well as the other new additions to this file?

Reply:
No it wasn't expected, thanks for pointing this out

Reply:
I just updated the PR